classes/Mission.php

```diff
@@ -43,6 +47,7 @@ class Mission {
     public $stages;
     public $money;
     public $rewards = [];
+    public ?TravelCoords $origin;
```

Comment:
```suggestion
    public ?TravelCoords $custom_start_location;
```
I was a little unsure reading this originally, I think a more specific name like this would help make it easier to understand how this field is used

Reply:
Fixed in commit.